autoscaler/controllers/gateway/config/root.go

```diff
@@ -38,7 +38,13 @@ func Calculate(dests *odigosv1.DestinationList, processors *odigosv1.ProcessorLi
 			return "", fmt.Errorf("no configer for %s", dest.Spec.Type)
 		}
 
-		configer.ModifyConfig(&dest, currentConfig)
+		err := configer.ModifyConfig(&dest, currentConfig)
+		if err != nil {
+			return "", err.Error()
+		} else {
+			return "Success", nil
```

Comment:
Notice that the function return value is a yaml representation of configmap. returning the string "Success" here is not compatible with how the value is used in the caller to `Calculate` function